wallet/network.go

```diff
@@ -122,6 +170,16 @@ func (o OfflineNetworkBackend) Synced(ctx context.Context) (bool, int32) {
 	return true, 0
 }
 
+func (o OfflineNetworkBackend) Done() <-chan struct{} {
+	c := make(chan struct{})
```

Comment:
Rather than inefficiently creating a new channel on every invocation of `Done`, which happens frequently, wouldn't it be better to just create the closed channel when the `OfflineNetworkBackend` itself is created, store it as a member in the struct, and return it here?